### ISSUE TITLE
 MelGAN-Pytorch Visulization Problem

### ISSUE THREAD

**Issue by Haohan-SHI** [jaywalnut310/MelGAN-Pytorch#4]
Hi, 

I am a Ph.D. student from the UK, and I am implementing your code 'MelGAN-Pytorch', after the training process I don't know how to visualize the result, for example, to plot the training & validation loss of both the Generator and Discriminator. Can you please help me with it? Many thanks!

Best wishes,
Haohan
